lib/linter/code-path-analysis/code-path-segment.js

```diff
@@ -88,7 +88,7 @@ class CodePathSegment {
             }
         });
 
-        /* istanbul ignore if */
+        /* c8 ignore if */
         if (debug.enabled) {
             this.internal.nodes = [];
         }
```

Comment:
`c8` shouldn't understand whether there is an `if` statement or not, as it does not parse the AST. There is no such option as `c8 ignore if`.

```suggestion
        /* c8 ignore next 3 */
        if (debug.enabled) {
            this.internal.nodes = [];
        }
```

Reply:
thanks, I updated it to use `c8 ignore start/end` - in case it may have > 3 statements in future changes. :)

Reply:
This same comment applies to `c8 ignore else` comments. There is one still remaining. 😄 

See https://github.com/bcoe/c8/#ignoring-uncovered-lines-functions-and-blocks for all options.